index.html

```diff
@@ -24,6 +24,16 @@
               <li><a href="#contact-section-link">Contact</a></li>
             </ul>
         </nav>
+
+        <div class="mobile-menu">
+          <img class="menu-x" src="images/icon-menu-x.png" alt="menu x">
+          <ul class="mobile-menu-ul">
+            <li><a class="mobile-menu-option" href="#portfolio-section-link">Portfolio</a></li>
+            <li><a class="mobile-menu-option" href="#about-section-link">About</a></li>
+            <li><a class="mobile-menu-option" href="#contact-section-link">Contact</a></li>
+          </ul>
+        </div> 
+
```

Comment:
- [ ] Great work adding the mobile menu. Almost all features have been met but one. Please add a[ blur](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/blur) to your mobile menu so that the menu items standout above the texts in the background. 
**Figma**
![blur](https://github.com/JavierAybar/Portfolio/assets/99656687/7ab42159-5e3d-4729-a3cb-046ebf2df875)


**Your design**
![blur](https://github.com/JavierAybar/Portfolio/assets/99656687/a7b06afe-b411-4bb8-818a-9445fb6dfe0f)

Reply:
hello in my style i have the property backdrop-filter: blur(0.5rem); in the class .mobile-menu

Reply:
@wayungi